18-hlt-intro.md

```diff
@@ -60,54 +69,56 @@ report = reports.decReport('Hlt1TrackAllL0Decision')
 print report.decision()
 ~~~
 
-The HLT1 selection that was most efficient for hadronic charm and beauty decays
-in Run-I was called Hlt1TrackAllL0. Use the advance function to find an event
-that was accepted by that trigger selection.
+The HLT1 selections that are most efficient for hadronic charm and beauty decays
+in Run-II are called Hlt1TrackMVA and Hlt1TwoTrackMVA. Use the advance function
+to find an event that was accepted by either of these trigger selections.
 
 The DecReports only contain the decisions, the candidates are stored in the
 SelReports ("Hlt{1,2}/SelReports"). Get the Hlt1 SelReports from the event store
-and retrieve the one for Hlt1TrackAllL0 using the selReport function,
+and retrieve the one for one of the TrackMVA selections using the selReport function,
 analogously to how the DecReport was retrieved above.
 
 The SelReports store candidates in a tree of sub-structures, which can be
 accessed using the substructure member funtion of a report. Any SelReport has at
 least one level of sub-structure. The sub-structure is internally stored in
 SmartRefs, which can be derefereced using their "data" method. Let's have a look
-at a SelReport for Hlt1TrackAllL0.
+at a SelReport for an TrackMVA selection.
 
 ~~~ {.python}
 reports = evt['Hlt1/SelReports']
-report = reports.selReport('Hlt1TrackAllL0Decision')
+report = reports.selReport('Hlt1TrackMVADecision')
 print report
 report.substructure().size()
-report.substructure().substructure()[0]
-report.substructure().substructure()[0].data()
+report.substructure()[0].substructure().size()
+report.substructure()[0].substructure()[0]
+report.substructure()[0].substructure()[0].data()
 ~~~
 
 In addition to the LHCbIDs, some numbers are also stored, such as the momentum
 of the track. These are stored in the numerical info dictionary that can be
 retrieved using:
```

Comment:
Again, we haven't explicitly described `LHCbID` objects up until this point. Would be worth another callout box. Maybe something like:


```
> ## What is an LHCbID? {.callout}
> Every single sub-detector element has an LHCbID which is unique across the whole detector.
> Physics objects, such as tracks, can be defined as sets of LHCbID objects.
> When a trigger decision is made, the set of LHCbID objects which comprise the triggering
> object is stored in the SelReports. This allows objects reconstructed later, such as in Brunel,
> to be compared with the objects reconstructed in the trigger.
```

The last part may be unnecessary given the TIS/TOS.